lib/e_chronicler_web/templates/journal_entry/form.html.heex

```diff
@@ -0,0 +1,23 @@
+<.form let={f} for={@changeset} action={@action}>
+  <%= if @changeset.action do %>
+    <div class="alert alert-danger">
+      <p>Oops, something went wrong! Please check the errors below.</p>
+    </div>
+  <% end %>
+
+  <%= label f, :title %>
+  <%= text_input f, :title %>
+  <%= error_tag f, :title %>
+
+  <%= label f, :author %>
+  <%= text_input f, :author %>
+  <%= error_tag f, :author %>
+
+  <%= label f, :entry %>
+  <%= textarea f, :entry %>
+  <%= error_tag f, :entry %>
+
+  <div >
+    <%=  submit "Save" %>
+  </div>
```

Comment:
I ended up removing the `div` because the Phoenix form automatically renders a button and adding more button tags makes a weird little extra vestigial button onto the existing button.